assets/locales/core.en.toml

```diff
@@ -688,3 +688,43 @@ one = "December"
 [TableScrollableDefaultAriaLabel]
 description = "Scrollable table"
 one = "Scrollable table"
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+# ERROR PAGES
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+[401Title]
```

Comment:
Bit of scope creep but do you think we should include a 400 - client error? I know the [search controller](https://github.com/ONSdigital/dp-frontend-search-controller/blob/e73230d93163dcca97747a3546b9421cd5386397/handlers/handlers.go#L403) and the [census filter/flex controller](https://github.com/ONSdigital/dp-frontend-filter-flex-dataset/blob/3e2d2f7e0dc542fffb3e50f5ca6be161b95cb4be/handlers/common.go#L73) can return it

Reply:
Definitely scope creep but still a valid point. Hopefully this work is extensible so one could be added easily in future. 

We don't render one anywhere currently -unless i'm missing it? - so i don't think we'll lose any functionality by not including it right now. 

Potentially we'd want to UX/content design it first anyways? And what's causing the 400? If its user input we might want to consider a better approach than a full error page maybe something on the form that's been submitted? Just a thought